disnake/interactions/modal.py

```diff
@@ -134,19 +142,18 @@ class ModalInteractionData(Dict[str, Any]):
     ----------
     custom_id: :class:`str`
         The custom ID of the modal.
+    components: List[:class:`dict`]
+        The raw component data of the modal interaction.
+
+        .. versionadded:: 2.6
     """
 
-    __slots__ = ("custom_id", "_components")
+    __slots__ = ("custom_id", "components")
 
     def __init__(self, *, data: ModalInteractionDataPayload):
         super().__init__(data)
         self.custom_id: str = data["custom_id"]
-        # this attribute is not meant to be used since it lacks most of the component data
```

Comment:
we should keep a portion of this comment; the data is still incomplete.

Reply:
acca2ae01f712587e6d0c111b868801e49502c60